package.json

```
@@ -40,7 +40,8 @@
     "lint:fix:scss": "stylelint \"**/*.scss\" --fix",
     "fmt": "prettier --write .",
     "fmt:check": "prettier --check .",
-    "start": "npm-run-all build:sass --parallel *:*:watch"
+    "start": "npm-run-all build:sass --parallel *:*:watch",
+    "init-blog-dates": "node tools/update-blog-dates.js --init"
```

To follow our conventions:
```suggestion
    "build:init-blog-dates": "node tools/update-blog-dates.js --init"
```